### ISSUE TITLE
Header height based on Orientation

### ISSUE THREAD

**Issue by kumaresan-cgvak** [react-native-elements/react-native-elements#1791]
**Is your feature request related to a problem? Please Describe.**
Header height is not changed according to the orientation of mobile device. So, header occupies more space in landscape mode.

**Describe the solution you'd like**
Header height can be reduced in Landscape mode of a mobile device.

**Comment by dquessenberry** [comment]
Not sure why this issue was closed as it is a legit problem. I am able to reproduce the issue and I've looked through the Header source code, and it does not resize the header height for "notched" iPhones appropriately on orientation change. If the app is refreshed in landscape it does render the Header height appropriately, so I assume the Header component just needs an orientation change listener and to resize the height accordingly from portrait to landscape and vice versa.

In the meantime, I am writing a wrapper component to handle this, but it would be nice if the Header component did this automatically.
